src/test/java/org/mockitousage/spies/SpyCreationShouldNotAffectMockTest.java

```diff
@@ -1,48 +0,0 @@
-/*
- * Copyright (c) 2007 Mockito contributors
- * This program is made available under the terms of the MIT License.
- */
-package org.mockitousage.spies;
-
-import org.junit.Test;
-import org.mockitoutil.TestBase;
-
-import static org.junit.Assert.assertEquals;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.spy;
-
-public class SpyCreationShouldNotAffectMockTest extends TestBase {
```

Comment:
Same here

Reply:
I do not think these tests make sense as things currently are in Mockito. They basically test for the feature that is removed. I think there's a better way for test for this once we choose to enable hashCode/equals enablement.